odom_covariance_override:
  ros__parameters:
    publish_topic_name:   odom      # publishするトピック名
    subscribe_topic_name: odom/raw  # subscribeするトピック名
    override_frame_id: false        # frame_idを上書きするか
    frame_id:          odom         # publishするトピックのframe_id
    pose_covariance:  [ 0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
    twist_covariance: [ 0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]

    pose_covariance_magnification: 0.5      # pose position covarianceの倍率
    distance_threshold: 0.1                 # 前回からの移動量。この設定値以下の移動量だった場合、pose yaw covarianceは問答無用で2pi^2となる\
